Cargo.toml

```diff
@@ -7,9 +7,9 @@ authors = ["Alexey Galakhov"]
 license = "MIT/Apache-2.0"
 readme = "README.md"
 homepage = "https://github.com/snapview/tungstenite-rs"
-documentation = "https://docs.rs/tungstenite/0.5.3"
+documentation = "https://docs.rs/tungstenite/0.5.4"
 repository = "https://github.com/snapview/tungstenite-rs"
-version = "0.5.3"
+version = "0.5.4"
```

Comment:
The changes in this PR are not backward-compatible with the previous version (see the error types returned by some of the functions for instance), so I think we cannot just change a patch version in this case according to Semantic Versioning paradigm.